.github/workflows/test.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       - name: Install Foundry
         uses: foundry-rs/foundry-toolchain@v1
       - name: Install dependencies
-        run: yarn install --frozen-lockfile
+        run: yarn install --frozen-lockfile --network-concurrency 2
```

Comment:
With the introduction of the extra packages introduced by the seaport dependency`yarn install` was constantly failing on CI. Thanks to @timm088 for finding the workaround